widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Base.kt

```diff
@@ -19,6 +19,7 @@ internal fun DefaultTheme(pallet: ColorPallet?): UnifiedTheme? = pallet?.let {
         secureConversationsWelcomeScreenTheme = SecureConversationsWelcomeScreenTheme(it),
         secureConversationsConfirmationScreenTheme = SecureConversationsConfirmationScreenTheme(it),
         snackBarTheme = DefaultSnackBarTheme(it),
-        webBrowserTheme = WebBrowserTheme(it)
+        webBrowserTheme = WebBrowserTheme(it),
+        entryWidgetTheme = DefaultEntryWidgetTheme(it)
```

Comment:
Out of interest, why some themes are called 'Default' and others aren't? For example, `DefaultEntryWidgetTheme` and `CallVisualizerTheme`.

Reply:
I think we need to align them

Reply:
For now we can rename `DefaultEntryWidgetTheme` to `EntryWidgetTheme`, at least. As a result, we would have only one name that stands out - `DefaultSnackBarTheme`.

Reply:
@DavDo Do you want me to rename the `DefaultEntryWidgetTheme`?

Reply:
Yes, please.
For the snack bar, I think we don't have alternatives since we also have a `SnackBarTheme.`

Reply:
I think @gugalo should have taken this PR.